ssc/cmod_pvsamv1_eqns.cpp

```diff
@@ -0,0 +1,354 @@
+#include "cmod_pvsamv1_eqns.h"
+
+#include "vartab.h"
+
```

Comment:
Are these specific to the ReOpt calls or used throughout the UI?

Reply:
so far there's just this one equation that's only used in the sam/src/invoke.cpp::fcall_reopt_battery_sizing